code_generator/GdkCodeGenerator/src/Generation/Model/UnitySchemaFile.cs

```diff
@@ -163,6 +163,7 @@ public class UnityCommandDefinition
             internal readonly TypeReferenceRaw RawResponseType;
             public UnityTypeReference RequestType;
             public UnityTypeReference ResponseType;
+            public uint CommandIndex;
```

Comment:
While you are at it, could you also add evenIndex fields?

Reply:
Will need to make a PR to `develop-code-generator` and rebuild the `dll` for that I think - but should be easy 😄

Reply:
Thanks and sorry for the hassle. This just came to my mind because the monobehaviours would benefit from an eventindex. Doesn't need to to be part of PR. :)

Reply:
Opened a ticket for this